projects/front_power_distribution/test/test_front_power_distribution_gpio.c

```diff
@@ -0,0 +1,53 @@
+#include "front_power_distribution_events.h"
+#include "front_power_distribution_gpio.h"
+#include "log.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+static GpioAddress s_pin_addresses[] = {
```

Comment:
it's not a bad idea to expose a `GpioAddress * test_provide_gpio_addresses()`to your `front_power_distribution_gpio` module, that simply returns `s_output_gpio_pins`. That way you don't have to redefine everything here again.

Reply:
`s_output_gpio_pins` is indexed by `FrontPowerDistributionGpioOutput`s, not `FrontPowerDistributionGpioEvent`s, so I'd have to expose `s_events_to_gpio_outputs` in a similar function too. Should I do that?